CHANGELOG.md

```diff
@@ -5,40 +5,56 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#45)
+- Removed unnecessary `Unit` argument in creation functions for `HTMLImageElement` and `HTMLAudioElement` (#46)
 
 New features:
+- Added `HTMLHyperlinkElementUtils` module (#44)
+- Added `create` and `create'` functions for `Audio` (#32)
+- Added missing attributes for `HTMLImage`, including `srcset`, `currentSrc`, `sizes`, `referrerPolicy`, `decoding`, `loading` (#40)
+- Added common `PropName`, `AttrName`, and `ClassName` which can be reused across libraries representing properties, attributes, and CSS classes (#30)
 
 Bugfixes:
+- Fixed return type of `scrollX` and `scrollY` to be `Effect Number`, as these can return subpixel values (#17)
+- Removed `Newtype` instances for types with hidden constructors (#39)
```

Comment:
Shouldn't this be a breaking change since one might have been depending on a type having a `Newtype` instance?